fgrpc/grpcrunner_test.go

```diff
@@ -95,6 +95,16 @@ func TestGRPCDestination(t *testing.T) {
 			"localhost:1234",
 			"localhost:1234",
 		},
+		{
+			"http hostname and port",
+			"http://localhost:1234",
+			"localhost:1234",
+		},
+		{
+			"https hostname and port",
+			"https://localhost:1234",
+			"localhost:1234",
+		},
 		{
 			"IPv4 address",
 			"1.2.3.4",
```

Comment:
can you add tests for http:// and https://foo.bar:2567 -> foo.bar:2567 ?

Reply:
done